### DISCUSSION TITLE
Project fields don't seem available for Issues when writing workflows.

### DISCUSSION THREAD

**Discussion by RobertIsmo** [community/community#43154] [Actions] [answered]
### Select Topic Area

Question

### Body

Was wondering if this is intentional.

[The documentation for the issue object](https://docs.github.com/en/graphql/reference/objects#issue) states that issues have fields for projects associated with the issue(ex. projectCards, projectItems, ProjectsV2).

however when building an action that uses these it seems that they don't exist. I printed out the github.event.issue and it seems none of these project fields are available.Is it because these fields can circularly reference the issue so they can't actually be serialized. If so how would one check if someone assigned an issue to a project on creation?

**Comment by Simran-B** [comment, marked as answer]
The documentation you linked to is about the GraphQL API. However, `github.event` gives you the webhook event payload of the REST API:
- https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues (`issues` link under **Webhook event payload**)
- https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#issues (in the **Body parameters** list under **`issue` object**, expand the **Properties of `issue`**)

My understanding is that you need to use the GraphQL API manually to access the information, e.g. by using `gh api graphql -f query='...'` in a step's `run` code ([source](https://liblab.com/blog/using-github-graphql-api-with-github-actions)), or by using the `actions/github-script` action: https://github.com/actions/github-script#run-custom-graphql-queries

**Reply by bossanovaorca** [reply]
based on those docs, it doesn't seem possible to trigger workflows based on changes in a project item's fields, is that right?

**Reply by Simran-B** [reply]
I believe you are right as all the `project*` events have a remark

> Note: This event only occurs for projects (classic).
